package.json

```diff
@@ -26,7 +26,7 @@
     "react-intl": "^2.8.0"
   },
   "optionalDependencies": {
-    "chromedriver": "77.0.0"
+    "chromedriver": "79.0.0"
```

Comment:
Since we always update to the latest, should we just make it `>= 79`?

Reply:
I think this is not really better, as the relevant version is (AFAIK) the chromium version of EPEL, which seems not to update every version.